sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,714 @@
+# Azure Communication Chat Service client library for Android
+This package contains the Android SDK for Azure Communication Services for Chat.
+Read more about Azure Communication Services [here](https://docs.microsoft.com/azure/communication-services/overview).
+
+# Getting started
+
+## Prerequisites
+
+* You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this library.
+* An Azure Communication Resource, learn how to create one from [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource).
+* The client libraries natively target Android API level 21. Your application's minSdkVersion must be set to 21 or higher to use this library.
+* The library is written in Java 8. Your application must be built with Android Gradle Plugin 3.0.0 or later, and must be configured to [enable Java 8 language desugaring](https://developer.android.com/studio/write/java8-support.html#supported_features) to use this library. Java 8 language features that require a target API level > 21 are not used, nor are any Java 8+ APIs that would require the Java 8+ API desugaring provided by Android Gradle plugin 4.0.0.
+
+### Versions available
+The current version of this library is **1.0.0-beta.1**.
```

Comment:
If the current version of the library is this one then why does the CHANGELOG show `1.0.0-beta.2`?

Reply:
For this one, I check with Reza and we want the version labels for all communication packages to be consistent. Therefore, beta.2 is the correct one to use.